neuralpp/experiments/experimental_inference/belief_propagation.py

```diff
@@ -0,0 +1,29 @@
+from typing import List, Optional
```

Comment:
It looks like you did an amend rather than create a new commit (I only see one). For the future please create a new commit so I can see what the recent changes were specifically.